File: ex03/ex03_05/ex03_05.py
def main():
  x = int(input("Enter x: "))
  y = int(input("Enter y: "))
  new_xy = x & y
  print(str(x) + " & " + str(y) + " = " + str(new_xy))

  new_xy = x | y
  print(str(x) + " | " + str(y) + " = " + str(new_xy))

  new_xy = x ^ y
  print(str(x) + " ^ " + str(y) + " = " + str(new_xy))
  return 0

if __name__ == "__main__":
  main()

File: ex03/ex03_05/ex03_05.c
#include <stdio.h>

int main() {
  int x = 0;
  int y = 0;
  int new_xy = 0;

  printf("Enter x: ");
  scanf("%d", &x);
  printf("Enter y: ");
  scanf("%d", &y);
  new_xy = x & y;
  printf("%d & %d = %d\n", x, y, new_xy);

  new_xy = x | y;
  printf("%d | %d = %d\n", x, y, new_xy);

  new_xy = x ^ y;
  printf("%d ^ %d = %d\n", x, y, new_xy);

  return 0;
}